geonetwork/transform-mcp-iso19115-3/src/test/java/au/org/aodn/xslt/TransformCatalogueTest.java

```diff
@@ -71,7 +76,7 @@ public void testAllMetadataTags() throws IOException {
         for (File testCaseDir: testDir.toFile().listFiles(File::isDirectory)) {
             String testCaseName = testCaseDir.getName();
             Path expectedFile = resourceDir.resolve("expected").resolve(testCaseName + ".xml");
-            Path actualFile = testCaseDir.toPath().resolve("metadata/metadata.iso19115-3.2018.xml");
+            Path actualFile = testCaseDir.toPath().resolve("metadata/2d2b2f92-12fa-4330-a480-94f0892c2b72.xml");
```

Comment:
So this is wierd - I pyulled down latest changes to the branch locally - and this one has been fixed.  github issues?

Reply:
And here

Reply:
Fixed. Removed hardcoded uuid